docs/api/networksocket/CellularNonIPSocket.md

```diff
@@ -0,0 +1,83 @@
+## Non-IP cellular socket
+
+The CellularNonIPSocket class provides, through standard socket send and recv member functions, the ability to send and receive Non-IP datagrams using Control Plane CIoT EPS optimisation feature of the NB-IoT networks. This feature is implemented in ControlPlane_netif class. [`ControlPlane_netif`](https://os.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_control_plane__netif.html) class.
```

Comment:
State here "send and receive the 3GPP Non-IP datagrams (NIDD)"

Reply:
Remove the Control Plane CIoT EPS --- scary :) This could be removed in many places, just say "Cellular IoT" ?

Reply:
done